src/containers/Tenant/Diagnostics/Network/Network.scss

```diff
@@ -111,7 +111,7 @@
         display: flex;
         gap: 12px;
 
-        margin-bottom: 16px;
+        margin: 0 16px 16px 0;
```

Comment:
The right margin on empty lists was forced by the image, now that it became smaller, the layout squashed. This fixes it.